zipkin/src/main/java/com/linecorp/armeria/common/tracing/RequestContextCurrentTraceContext.java

```diff
@@ -69,6 +78,28 @@ public String toString() {
         }
     };
 
+    /**
+     * Trace context scopes are often wrapped, for example to decorate MDC. This creates a dummy scope to
+     * ensure {@link RequestContextCurrentTraceContext#INSTANCE} is configured, even if it is wrapped.
```

Comment:
Now that this method is part of public API, it needs to be more user-centric. How about:

```
Ensures the specified {@link Tracing} uses a {@link RequestContextCurrentTraceContext}.

@throws IllegalStateException if {@code tracing} does not use {@link RequestContextCurrentTraceContext}
```

Reply:
ack